air-script/tests/main.rs

```diff
@@ -155,6 +155,7 @@ fn list_folding() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Sorry if I missed this explanation somewhere - why is this test being ignored?

Reply:
It uses the `match` syntax which was removed, a follow-on PR will re-add it with the syntax changes we discussed and the test will be re-enabled at that time